app/src/main/java/com/bogdan/codeforceswatcher/components/compose/textfields/HandleTextField.kt

```diff
@@ -11,7 +12,8 @@ import com.bogdan.codeforceswatcher.R
 
 @Composable
 fun HandleTextField(
-    onValueChange: (String) -> Unit
+    modifier: Modifier = Modifier,
```

Comment:
The modifier should always be the first optional parameter. All non-optional params should be first.